component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -288,6 +288,14 @@
                         optional = true,
                         type = {DataType.STRING},
                         defaultValue = "None"
+                ),
+                @Parameter(
+                        name = Constants.FILE_NAME_PATTERN,
+                        description = "Regex pattern for the filenames that should be read from the directory. " +
+                                "Note: This parameter is applicable only if the connector is reading from a directory",
+                        optional = true,
+                        type = {DataType.STRING},
+                        defaultValue = "None"
```

Comment:
@CharukaK shall we use defaultValue = "<Empty_String>" ?

Reply:
fixed